sklearn/cross_validation.py

```diff
@@ -973,7 +973,7 @@ def _index_param_value(X, v, indices):
     return safe_indexing(v, indices)
 
 
-def cross_val_predict(estimator, X, y=None, cv=None, n_jobs=1,
+def cross_val_apply(estimator, X, y=None, cv=None, n_jobs=1, decision_func='predict',
```

Comment:
This shouldn't be called ``decision_func`` I think. Maybe ``predict_func`` or ``apply_func``?